### DISCUSSION TITLE
implementing TRPC support

### DISCUSSION THREAD

**Discussion by DasMergo** [payloadcms/payload#4061] [Feature Requests & Ideas]
i think integrating TRPC with Payload CMS may require custom backend development to bridge the gap between TRPC and the CMS.
it seems that Payload CMS is considering moving from Express to Next.js so it would be nice to have TRPC support in payloadcms because tRPC provides a seamless developer experience when used with Next.js.

**Comment by GermanJablo** [comment]
tRPC can be used seamlessly with Payload, and I don't think there's any additional integration or anything else we need to do on our end.
Also as a curiosity, [our next sdk](https://github.com/payloadcms/payload/pull/9463) is going to solve a somewhat related problem.
